### PR TITLE
Fixing cs issues in Test folder

### PR DESCRIPTION
**Pull request by simon-letch**

### CODE REVIEW

File: Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php [hunk 1]
```diff
@@ -8,9 +8,20 @@ class HistoricalCatalogDataRefreshTest extends \Magento\TestFramework\TestCase\A
      * @var \Magento\Framework\ObjectManagerInterface
      */
     public $objectManager;
-    public $model = 'Dotdigitalgroup\Email\Model\Catalog';
+
+    /**
+     * @var [type]
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php]: [type] need to be string

File: Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php [hunk 2]
```diff
@@ -25,6 +36,12 @@ public function setUp()
         $this->getRequest()->setParams($params);
     }
 
+    /**
+     * @param mixed $from
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php]: All three params are of type String

File: Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php [hunk 3]
```diff
@@ -178,15 +213,22 @@ public function test_catalog_full_reset_success_with_to_date_only()
         $this->assertEquals(1, $collection->getSize());
     }
 
+    /**
+     * @param mixed $data
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalCatalogDataRefreshTest.php]: type is Array

File: Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php [hunk 4]
```diff
@@ -8,9 +8,20 @@ class HistoricalOrderDataRefreshTest extends \Magento\TestFramework\TestCase\Abs
      * @var \Magento\Framework\ObjectManagerInterface
      */
     public $objectManager;
-    public $model = 'Dotdigitalgroup\Email\Model\Order';
+
+    /**
+     * @var object
+     */
+    public $model = Dotdigitalgroup\Email\Model\Order::class;
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php]: actually type is String ::class returns String

File: Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php [hunk 5]
```diff
@@ -8,9 +8,20 @@ class HistoricalOrderDataRefreshTest extends \Magento\TestFramework\TestCase\Abs
...
+
+    /**
+     * @var object
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php]: type is String

File: Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php [hunk 6]
```diff
@@ -25,6 +36,12 @@ public function setUp()
         $this->getRequest()->setParams($params);
     }
 
+    /**
+     * @param mixed $from
+     * @param mixed $to
+     * @param mixed $dispatchUrl
+     * @return void
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php]: all three are of type String

File: Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php [hunk 7]
```diff
@@ -201,15 +236,22 @@ public function test_order_full_reset_success_with_to_date_only()
         $this->assertEquals(1, $collection->getSize());
     }
 
+    /**
+     * @param mixed $data
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalOrderDataRefreshTest.php]: $data type is array

File: Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php [hunk 8]
```diff
@@ -8,9 +8,20 @@ class HistoricalReviewDataRefreshTest extends \Magento\TestFramework\TestCase\Ab
      * @var \Magento\Framework\ObjectManagerInterface
      */
     public $objectManager;
-    public $model = 'Dotdigitalgroup\Email\Model\Review';
+
+    /**
+     * @var object
+     */
+    public $model = Dotdigitalgroup\Email\Model\Review::class;
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php]: type is String

File: Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php [hunk 9]
```diff
@@ -8,9 +8,20 @@ class HistoricalReviewDataRefreshTest extends \Magento\TestFramework\TestCase\Ab
...
+
+    /**
+     * @var object
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php]: type is String

File: Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php [hunk 10]
```diff
@@ -25,6 +36,12 @@ public function setUp()
         $this->getRequest()->setParams($params);
     }
 
+    /**
+     * @param mixed $from
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php]: All three params type is String

File: Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php [hunk 11]
```diff
@@ -192,15 +227,22 @@ public function test_review_full_reset_success_with_to_date_only()
         $this->assertEquals(1, $collection->getSize());
     }
 
+    /**
+     * @param mixed $data
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalReviewDataRefreshTest.php]: type is Array

File: Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php [hunk 12]
```diff
@@ -8,9 +8,20 @@ class HistoricalWishlistDataRefreshTest extends \Magento\TestFramework\TestCase\
      * @var \Magento\Framework\ObjectManagerInterface
      */
     public $objectManager;
-    public $model = 'Dotdigitalgroup\Email\Model\Wishlist';
+
+    /**
+     * @var object
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php]: type is String

File: Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php [hunk 13]
```diff
@@ -25,6 +36,12 @@ public function setUp()
         $this->getRequest()->setParams($params);
     }
 
+    /**
+     * @param mixed $from
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php]: All params are of type String

File: Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php [hunk 14]
```diff
@@ -199,15 +234,22 @@ public function test_wishlist_full_reset_success_with_to_date_only()
         $this->assertEquals(1, $collection->getSize());
     }
 
+    /**
+     * @param mixed $data
```
Comment (adeelq) [inline Test/Integration/Adminhtml/Developer/HistoricalWishlistDataRefreshTest.php]: type is Array

File: Test/Integration/Automation/FirstOrderTest.php [hunk 15]
```diff
@@ -57,10 +70,14 @@ public function test_first_customer_automation()
         $this->assertEquals(1, $automation->getSize(), 'duplicate automation for first order');
     }
 
+    /**
+     * @param mixed $order
```
Comment (adeelq) [inline Test/Integration/Automation/FirstOrderTest.php]: type is \Magento\Sales\Model\Order

File: Test/Integration/Helper/ApiEndpointTest.php [hunk 16]
```diff
@@ -37,12 +46,14 @@ public function removeData()
     }
 
     /**
-     * @param $website
-     * @param $endPoint
+     * @param mixed $website
```
Comment (adeelq) [inline Test/Integration/Helper/ApiEndpointTest.php]: type is String

File: Test/Integration/Helper/ApiEndpointTest.php [hunk 17]
```diff
@@ -37,12 +46,14 @@ public function removeData()
...
+     * @param mixed $endPoint
```
Comment (adeelq) [inline Test/Integration/Helper/ApiEndpointTest.php]: type is String

File: Test/Integration/Oberver/CreateUpdateContactTest.php [hunk 18]
```diff
@@ -106,6 +114,10 @@ public function test_contact_created_and_updated_successfully()
         $this->assertEquals($updatedEmail, $emailUpdated, 'Updated contact was not found');
     }
 
+    /**
+     * @param mixed $updatedEmail
```
Comment (adeelq) [inline Test/Integration/Oberver/CreateUpdateContactTest.php]: type is String

File: Test/Integration/Rules/RulesTest.php [hunk 19]
```diff
@@ -26,6 +26,10 @@ class RulesTest extends \PHPUnit_Framework_TestCase
 {
     const RULE_OPERATOR_AND = 1;
     const RULE_OPERATOR_OR = 2;
+
+    /**
+     * @var object
+     */
     public $serializer;
```
Comment (adeelq) [inline Test/Integration/Rules/RulesTest.php]: Seems like not used anywhere in test so it is unused

Comment (simon-letch) [inline Test/Integration/Rules/RulesTest.php]: Ln217?

**Comment by simon-letch** [comment]
@adeelq i've fixed all points you mentioned. Can you confirm?

**Comment by adeelq** [comment]
@simon-letch 9 tests failing

File: Test/Integration/Automation/FirstOrderTest.php [hunk 20]
```diff
@@ -57,10 +70,14 @@ public function test_first_customer_automation()
...
 
+    /**
+     * @param \Magento\Sales\Model\Order $order
+     * @return void
+     */
     public function createInvoice($order)
     {
         $orderService = \Magento\TestFramework\ObjectManager::getInstance()->create(
-            'Magento\Sales\Api\InvoiceManagementInterface'
+            Magento\Sales\Api\InvoiceManagementInterface::class
```
Comment (adeelq) [inline Test/Integration/Automation/FirstOrderTest.php]: needs to be \Magento\Sales\Api\InvoiceManagementInterface::class

File: Test/Integration/Helper/ApiEndpointTest.php [hunk 21]
```diff
@@ -12,23 +12,32 @@
...
+     * @return void
+     */
     public function removeData()
     {
         /** @var ObjectManager $objectManager */
         $objectManager = ObjectManager::getInstance();
 
         /** @var \Magento\Config\Model\ResourceModel\Config $config */
-        $config = $objectManager->create('Magento\Config\Model\ResourceModel\Config');
+        $config = $objectManager->create(Magento\Config\Model\ResourceModel\Config::class);
```
Comment (adeelq) [inline Test/Integration/Helper/ApiEndpointTest.php]: needs to be \Magento\Config\Model\ResourceModel\Config::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 22]
```diff
@@ -62,17 +70,30 @@ class OrderSyncTest extends \Magento\TestFramework\TestCase\AbstractController
...
-        $this->helper = $this->getMock('Dotdigitalgroup\Email\Helper\Data', [], [], '', false);
+        $this->connectorOrderFactory = $this->getMock(
+            'Dotdigitalgroup\Email\Model\Connector\OrderFactory',
+            [],
+            [],
+            '',
+            false
+        );
+        $this->contactResource = ObjectManager::getInstance()->get(
+            Dotdigitalgroup\Email\Model\ResourceModel\Contact::class
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: need to be \Dotdigitalgroup\Email\Model\ResourceModel\Contact::class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 23]
```diff
@@ -14,34 +14,43 @@ class SingleOrderSyncTest extends \PHPUnit_Framework_TestCase
...
 
+    /**
+     * @return void
+     */
     public function setup()
     {
         $this->objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
         $this->importerCollection = $this->objectManager->create(
-            'Dotdigitalgroup\Email\Model\ResourceModel\Importer\Collection'
+            Dotdigitalgroup\Email\Model\ResourceModel\Importer\Collection::class
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: need to be \Dotdigitalgroup\Email\Model\ResourceModel\Importer\Collection::class

File: Test/Integration/Helper/ApiEndpointTest.php [hunk 24]
```diff
@@ -37,12 +46,14 @@ public function removeData()
     }
 
     /**
-     * @param $website
-     * @param $endPoint
+     * @param string $website
```
Comment (cdiacon) [inline Test/Integration/Helper/ApiEndpointTest.php]: website should be  int

File: Test/Integration/Rules/RulesTest.php (hunk 19 above)
Comment (cdiacon) [inline Test/Integration/Rules/RulesTest.php]: indeed not used anywhere

**Comment by simon-letch** [comment]
Updated

**Comment by cdiacon** [comment]
@simon-letch in case you missed these ones 
https://github.com/dotmailer/dotmailer-magento2-extension/pull/422/files#diff-e6545d52aacdb96d66af9723fc80375cR263
https://github.com/dotmailer/dotmailer-magento2-extension/pull/422/files#diff-e6545d52aacdb96d66af9723fc80375cR263

File: Test/Integration/Sync/OrderSyncTest.php [hunk 25]
```diff
@@ -170,19 +205,23 @@ protected function prepareSync($withGuests = 0)
         /** @var ObjectManager $objectManager */
         $objectManager = ObjectManager::getInstance();
 
-        $website = $objectManager->create('Magento\Store\Model\Website');
-        $website->setData(['code' => 'test', 'name' => 'Test Website', 'default_group_id' => '1', 'is_default' => '0']);
+        $website = $objectManager->create(Magento\Store\Model\Website::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: need to be \Magento\Store\Model\Website::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 26]
```diff
@@ -170,19 +205,23 @@ protected function prepareSync($withGuests = 0)
...
+        $website->setData(
+            ['code' => 'test', 'name' => 'Test Website', 'default_group_id' => '1', 'is_default' => '0']
+        );
         $website->save();
         /** @var $store \Magento\Store\Model\Store */
-        $store = $objectManager->create('Magento\Store\Model\Store');
-        $store->setData([
+        $store = $objectManager->create(Magento\Store\Model\Store::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: \Magento\Store\Model\Store::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 27]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
         $this->helper->method('getWebsiteConfig')->willReturn('2');
         $this->helper->method('getConfigSelectedStatus')->willReturn($statuses);
 
-        $this->importerFactory->method('create')->willReturn($this->getMock('Dotdigitalgroup\Email\Model\Importer', ['registerQueue'], [], '', false));
+        $this->importerFactory->method('create')->willReturn(
+            $this->getMock(Dotdigitalgroup\Email\Model\Importer::class, ['registerQueue'], [], '', false)
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\Importer::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 28]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
...
         $connectorEmailOrder->method('setOrderData')->willReturn(true);
         $this->connectorOrderFactory->method('create')->willReturn($connectorEmailOrder);
 
-        $emailOrderCollection = $objectManager->get('\Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection');
+        $emailOrderCollection = $objectManager->get(
+            \Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection::class
+        );
 
-        $orderCollection = $objectManager->get('Magento\Sales\Model\ResourceModel\Order\Collection');
+        $orderCollection = $objectManager->get(Magento\Sales\Model\ResourceModel\Order\Collection::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Magento\Sales\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 29]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
...
 
         $connectorOrder = $this->getMock(\Dotdigitalgroup\Email\Model\Order::class, [], [], '', false);
 
         if ($withGuests) {
-            $connectorOrder->method('getOrdersToImport')->willReturn($this->getMock('Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection', [], [], '', false));
+            $connectorOrder->method('getOrdersToImport')->willReturn($this->getMock(
+                Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection::class,
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 30]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
...
+                '',
+                false
+            ));
             $connectorOrder->method('getModifiedOrdersToImport')->willReturn($emailOrderCollection);
         } else {
 
             $connectorOrder->method('getOrdersToImport')->willReturn($emailOrderCollection);
-            $connectorOrder->method('getModifiedOrdersToImport')->willReturn($this->getMock('Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection', [], [], '', false));
+            $connectorOrder->method('getModifiedOrdersToImport')->willReturn($this->getMock(
+                Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection::class,
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 31]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
...
 
+    /**
+     * @return void
+     */
     public function createNewEmailOrder()
     {
         $objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
 
-        $order = $objectManager->create('Magento\Sales\Model\ResourceModel\Order\Collection');
+        $order = $objectManager->create(Magento\Sales\Model\ResourceModel\Order\Collection::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Magento\Sales\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 32]
```diff
@@ -196,42 +235,72 @@ protected function prepareSync($withGuests = 0)
...
         $order = $order->getFirstItem();
 
-        $emailOrder = $objectManager->create('Dotdigitalgroup\Email\Model\Order');
+        $emailOrder = $objectManager->create(Dotdigitalgroup\Email\Model\Order::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\Order::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 33]
```diff
@@ -242,12 +311,15 @@ public function createNewEmailOrder()
...
     }
 
+    /**
+     * @return void
+     */
     public function createModifiedEmailOrder()
     {
         $objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
-        $order = $objectManager->create('Magento\Sales\Model\ResourceModel\Order\Collection');
+        $order = $objectManager->create(Magento\Sales\Model\ResourceModel\Order\Collection::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Magento\Sales\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/OrderSyncTest.php [hunk 34]
```diff
@@ -242,12 +311,15 @@ public function createNewEmailOrder()
...
         $order = $order->getFirstItem();
-        $emailOrder = $objectManager->create('Dotdigitalgroup\Email\Model\Order');
+        $emailOrder = $objectManager->create(Dotdigitalgroup\Email\Model\Order::class);
```
Comment (adeelq) [inline Test/Integration/Sync/OrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\Order::class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 35]
```diff
@@ -14,34 +14,43 @@ class SingleOrderSyncTest extends \PHPUnit_Framework_TestCase
...
     }
 
+    /**
+     * @return void
+     */
     public function prep()
     {
         /** @var  \Magento\Store\Model\Store $store */
-        $store = $this->objectManager->create('Magento\Store\Model\Store');
+        $store = $this->objectManager->create(Magento\Store\Model\Store::class);
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: needs to be \Magento\Store\Model\Store::class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 36]
```diff
@@ -14,34 +14,43 @@ class SingleOrderSyncTest extends \PHPUnit_Framework_TestCase
...
         $store->load($this->storeId);
 
-        $helper = $this->getMock('Dotdigitalgroup\Email\Helper\Data', [], [], '', false);
+        $helper = $this->getMock(Dotdigitalgroup\Email\Helper\Data::class, [], [], '', false);
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Helper\Data::class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 37]
```diff
@@ -50,15 +59,15 @@ public function prep()
         $helper->method('getConfigSelectedStatus')->willReturn($this->orderStatus);
 
         $orderSync = new \Dotdigitalgroup\Email\Model\Sync\Order(
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\ImporterFactory'),
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\OrderFactory'),
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\Connector\AccountFactory'),
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\Connector\OrderFactory'),
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\ResourceModel\Contact'),
-            $this->objectManager->create('Dotdigitalgroup\Email\Model\ResourceModel\Order'),
+            $this->objectManager->create(Dotdigitalgroup\Email\Model\ImporterFactory::class),
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: all classes from line 62 ti line 70 needs to have \ in front of the class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 38]
```diff
@@ -103,17 +118,20 @@ public function test_singe_order_type_is_object()
...
     }
 
+    /**
+     * @return null
+     */
     public function createModifiedEmailOrder()
     {
         /** @var \Magento\Sales\Model\ResourceModel\Order\Collection $orderCollection */
-        $orderCollection = $this->objectManager->create('Magento\Sales\Model\ResourceModel\Order\Collection');
+        $orderCollection = $this->objectManager->create(Magento\Sales\Model\ResourceModel\Order\Collection::class);
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: needs to be \Magento\Sales\Model\ResourceModel\Order\Collection::class

File: Test/Integration/Sync/SingleOrderSyncTest.php [hunk 39]
```diff
@@ -103,17 +118,20 @@ public function test_singe_order_type_is_object()
...
         /** @var \Magento\Sales\Model\Order $order */
         $order = $orderCollection->getFirstItem();
 
         $this->storeId = $order->getStoreId();
         $this->orderStatus = [$order->getStatus()];
 
-        $emailOrder = $this->objectManager->create('Dotdigitalgroup\Email\Model\Order')
+        $emailOrder = $this->objectManager->create(Dotdigitalgroup\Email\Model\Order::class)
```
Comment (adeelq) [inline Test/Integration/Sync/SingleOrderSyncTest.php]: needs to be \Dotdigitalgroup\Email\Model\Order::class

**Pull request merged**